src/main.rs

```diff
@@ -26,6 +26,10 @@ struct Cli {
     )]
     require_device: bool,
 
+    #[cfg(target_os = "linux")]
+    #[clap(long, short = 'd', help = "The path of the video device to monitor")]
```

Comment:
Let's add Linux only, to avoid confusion for anyone looking at the code:

```suggestion
    #[clap(long, short = 'd', help = "The path of the video device to monitor (Linux only)")]
```